content/open-source/open-source.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Should this stay as "open source" and not "open-source"?

Reply:
While both are totally acceptable, I think the hyphen `-` should be used whenever it is used as an adjective preceding a noun (i.e., "This is an _open-source_ project", "_Open-source_ software"). In other cases I'd omit the hyphen (i.e., "_Open source_ is ...")